projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1615,6 +1721,33 @@ public function add_jetpack_social_og_images( $tags ) {
 		$opengraph_image = $this->get_social_opengraph_image( get_the_ID() );
 
 		if ( empty( $opengraph_image ) ) {
+			// If we do not have a SIG or attached image, but we have an image in post body
```

Comment:
Where would you put it instead? I added it here, because this function is the one that is hooked onto the `jetpack_open_graph_tags` filter, where the Jetpack OG images are retrieved. So here if we do not have a SIG or attached media image here, we check the current OG one that's coming from the post from Jetpack

Reply:
@pablinos I refactored it, how about like this?